scapy/contrib/automotive/uds_scan.py

```diff
@@ -224,8 +223,9 @@ def cleanup(_, configuration):
         try:
             configuration["tps"].stop()
             configuration["tps"] = None
-        except (AttributeError, KeyError) as e:
-            log_automotive.debug("Cleanup TP-Sender Error: %s", e)
+        except (AttributeError, KeyError):
+            pass
+            # log_automotive.debug("Cleanup TP-Sender Error: %s", e)
```

Comment:
Is this intended?

Reply:
Yes, this is intended. I wanted to cleanup the logging from this message.